src/main/groovy/com/twcable/grabbit/jcr/ProtoNodeDecorator.groovy

```diff
@@ -97,6 +97,10 @@ class ProtoNodeDecorator {
                 log.debug "Added mixin ${value.stringValue} for : ${node.name}."
             }
             else {
+                JcrNodeDecorator decoratedNode = new JcrNodeDecorator(node)
```

Comment:
This is what I was trying to say in other comment.. ProtoNodeDecorator to be passed to JcrDecorator and that should be doing write part